src/status.c

```diff
@@ -31,6 +31,14 @@
  */
 
 static char status_onbranch[SIZEOF_STR];
+static bool show_untracked_only=false;
+
+void
+open_status_view(struct view *prev, bool untracked_only, enum open_flags flags)
+{
+	show_untracked_only=untracked_only;
+	open_view(prev, &status_view, flags | OPEN_REFRESH);
```

Comment:
Is the `OPEN_REFRESH` serving a purpose? It was not there in the inline version.

Reply:
Yes, without it, if you open the status view from the Untracked changes line and then want to open it after with the s key to have the full content, then the content remains the Untracked only.

Reply:
It looks like it causes an assertion failure.